packages/http/httpx/kiota_http/httpx_request_adapter.py

```diff
@@ -601,12 +601,14 @@ def get_request_from_request_information(
 
         otel_attributes = {
             HTTP_REQUEST_METHOD: method.value,
-            "http.port": url.port,
             SERVER_ADDRESS: url.hostname,
             URL_SCHEME: url.scheme,
             "url.uri_template": request_info.url_template,
         }
 
+        if url.port is not None:
+            otel_attributes["http.port"] = url.port
```

Comment:
can you call a string conversion here please so we maintain the 3.8 compatibility?

Reply:
Done!